web-ui/styles/openclaw-structured.css

```
@@ -264,3 +264,128 @@
+
+.openclaw-card-list .card {
+    background: rgba(255, 255, 255, 0.76);
+}
+
+.openclaw-card-list .card-subtitle {
+    white-space: normal;
+    display: -webkit-box;
+    -webkit-box-orient: vertical;
+    -webkit-line-clamp: 2;
+    overflow-wrap: anywhere;
+    word-break: break-word;
```

Stylelint (17.12.0)

[error] Deprecated keyword "break-word" for property "word-break" (declaration-property-value-keyword-no-deprecated)

(declaration-property-value-keyword-no-deprecated)